public/index.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+
+require __DIR__ . '/../vendor/autoload.php';
+
+$BANDWIDTH_ACCOUNT_ID = getenv("BANDWIDTH_ACCOUNT_ID");
+$BANDWIDTH_API_USER = getenv("BANDWIDTH_API_USER");
+$BANDWIDTH_API_PASSWORD = getenv("BANDWIDTH_API_PASSWORD");
+$BANDWIDTH_VOICE_APPLICATION_ID = getenv("BANDWIDTH_VOICE_APPLICATION_ID");
+$BASE_URL = getenv("BASE_URL");
+$PERSONAL_NUMBER = getenv("PERSONAL_NUMBER");
+
```

Comment:
No PORT config? No worries if not, either add the PORT or remove it from the readme

Reply:
removed from readme - you declare the port when you run the app in the terminal `php -S localhost:5000 -t public`